source/Calamari/Integration/Iis/WebServerSevenSupport.cs

```diff
@@ -203,21 +200,20 @@ public bool ApplicationPoolExists(string applicationPool)
             return FindApplicationPool(applicationPool) != null;
         }
 
-        private void Execute(Action<ServerManager> action)
+        void Execute(Action<ServerManager> action)
         {
-            using (var serverManager = ServerManager.OpenRemote(Localhost))
```

Comment:
This method appears to fail in NetCore 
https://github.com/dotnet/core/issues/2691
That's fine really since we are only interacting with the localhost anyway so we dont need to use `OpenRemote`